package.json

```diff
@@ -142,6 +142,7 @@
     "supertest": "6.3.3",
     "ts-jest": "29.1.0",
     "typescript": "5.3.2",
+    "yalc": "1.0.0-pre.53",
```

Comment:
Why are we using a pre-releasing?

Reply:
because they are not in 1.0.0 yet it's the latest